### ISSUE TITLE
generate emails when someone logs in to admin cp

### ISSUE THREAD

**Issue by andrewjs18** [mybb/mybb#4088]
mybb should consider adding some code that'll generate an email when someone logs in to the admin cp.  most sites that handle customer data do this and include some information such as this in the email it sends out:

```
Dear Andrew,

Someone has recently logged into your website account.

Username : me
IP Address : 1.2.3.4
Hostname : me.com
Time : 2020-06-21 02:50 PM

If this is not you we highly recommend you change your details and inform a
staff member immediately.

You can modify the settings of this alert under your profile.

Regards,
website
```

**Comment by Eldenroot** [comment]
Lets discuss on MyBB community site, not? Have you been hacked (on your board)?

**Comment by Eldenroot** [comment]
Do not bloat the core, better to add this as a plugin...

**Comment by andrewjs18** [comment]
created a thread on the forum: https://community.mybb.com/thread-228327.html

**Comment by andrewjs18** [comment]
for now, @Sama34 turned this functionality into a plugin: https://github.com/Sama34/OUGC-Email-ACP-Logins

**Comment by effone** [comment]
I also prefer this to have as a plugin.

**Comment by JordanMussi** [comment]
Plugin is best for now instead of adding new features to 1.8 near the end of its development. Will target this for 1.9.

**Comment by Sama34** [comment]
I agree this should be coded in core for both the ACP and the front end and as such 1.9 would be reasonable.
